buf/registry/module/v1/label.proto

```diff
@@ -37,11 +37,8 @@ message Label {
   google.protobuf.Timestamp create_time = 2 [(buf.validate.field).required = true];
   // The last time the Label was updated on the BSR.
   google.protobuf.Timestamp update_time = 3 [(buf.validate.field).required = true];
-  // Whether or not the Label was archived.
-  //
-  // If a Label is archived, the API considers it to not exist and will not return archived
-  // Labels, except when calling LabelService.GetLabels.
-  bool archived = 4;
+  reserved 4;
+  reserved "archived";
```

Comment:
could adjust all the field numbers if we would rather break and fix the CLI and the BSR

Reply:
We don't need to reserve anything - this API hasn't been released yet